prow/tide/README.md

```diff
@@ -0,0 +1,131 @@
+# Tide
+
+Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
+component for managing a pool of PRs that match a given set of criteria.
+Automatically retesting PRs in the pool and merging them if they pass tests
+against the latest base branch commit.
+
+Configuration of Tide is located under
+[prow/config.yaml](https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml)
+file. It consists of two parts:
+
+  * **Specification of PR pools/criteria**:
+    A set of all PRs Tide keeps track of.
+    The PRs are periodically checked if they are ready to be merged.
+    Configured under `tide` key of the configuration file.
+
+  * **Specification of PR tests**:
+    A set of jobs that are run over each PR before it gets merged.
+    No job is required to be provided in which case it's enough a PR has all
+    relevant labels set. If a set of jobs is specified,
+    all must succeed (unless configured otherwise).
+    Configured under `presubmits` key of the configuration file.
+
+## PR pools
+
+The set of criteria is specified through the following collection of items:
+
+```yaml
+tide:
+  queries:                // List of queries
+  - repo:                   // org/repo
+    labels:                 // List of must have labels
+    missingLabels:          // List of can't have labels
+    excludedBranches:       // Ignore branches
+    includedBranches:       // Include-only branches
+    reviewApprovedRequired: // Review approved is must
+  sync_period:            // Sync jobs period
+  status_update_period:   // PR status update period
+  merge_method:           // Squash, rebase or just merge PR
```

Comment:
Is any normalization happening during startup or does this need to be "squash" instead of "Squash"

Reply:
I will rephrase it as `Set to squash, rebase or just merge PR`